### ISSUE TITLE
Switch from `Trunk` to `bevy run web`

### ISSUE THREAD

**Issue by janhohenheim** [TheBevyFlock/bevy_new_2d#297]
Blocked on https://github.com/TheBevyFlock/bevy_cli/issues/8

**Comment by TimJentzsch** [comment]
Note that it will likely be `bevy run web` as I switched from a flag to a subcommand in the prototype

**Comment by benfrankel** [comment]
I believe this should be unblocked now?

**Comment by TimJentzsch** [comment]
Yes, I think so too! Finishing the port will be my top priority now.
